dart/lib/src/sentry.dart

```diff
@@ -36,27 +36,19 @@ class Sentry {
   /// such as SentryFlutter.
   static Future<void> init(
     OptionsConfiguration optionsConfiguration, {
-    AppRunner appRunner,
-    SentryOptions options,
+    AppRunner? appRunner,
+    SentryOptions? options,
   }) async {
-    if (optionsConfiguration == null) {
-      throw ArgumentError('OptionsConfiguration is required.');
-    }
-
-    final sentryOptions = options ?? SentryOptions();
+    final sentryOptions = options ?? SentryOptions(dsn: '');
```

Comment:
```suggestion
    final sentryOptions = options ?? SentryOptions();
```

no DSN is != than empty DSN